adafruit_is31fl3741/adafruit_ledglasses.py

```diff
@@ -22,38 +22,65 @@
   https://github.com/adafruit/circuitpython/releases
 
 """
+from struct import unpack_from
 import adafruit_is31fl3741
 from . import IS31FL3741
 
 
 class Right_Ring:
     """The right eye ring of the LED glasses"""
 
-    ledmap = (
-        (287, 31, 30),  # 0
-        (278, 1, 0),  # 1
-        (273, 274, 275),  # 2
-        (282, 283, 284),  # 3
-        (270, 271, 272),  # 4
-        (27, 28, 29),  # 5
-        (23, 24, 25),  # 6
-        (276, 277, 22),  # 7
-        (20, 21, 26),  # 8
-        (50, 51, 56),  # 9
-        (80, 81, 86),  # 10
-        (110, 111, 116),  # 11
-        (140, 141, 146),  # 12
-        (170, 171, 176),  # 13
-        (200, 201, 206),  # 14
-        (230, 231, 236),  # 15
-        (260, 261, 266),  # 16
-        (348, 349, 262),  # 17
-        (233, 234, 235),  # 18
-        (237, 238, 239),  # 19
-        (339, 340, 232),  # 20
-        (327, 328, 329),  # 21
-        (305, 91, 90),  # 22
-        (296, 61, 60),  # 23
+    # ledmap = (
+    #    (287, 31, 30),  # 0
+    #    (278, 1, 0),  # 1
+    #    (273, 274, 275),  # 2
+    #    (282, 283, 284),  # 3
+    #    (270, 271, 272),  # 4
+    #    (27, 28, 29),  # 5
+    #    (23, 24, 25),  # 6
+    #    (276, 277, 22),  # 7
+    #    (20, 21, 26),  # 8
+    #    (50, 51, 56),  # 9
+    #    (80, 81, 86),  # 10
+    #    (110, 111, 116),  # 11
+    #    (140, 141, 146),  # 12
+    #    (170, 171, 176),  # 13
+    #    (200, 201, 206),  # 14
+    #    (230, 231, 236),  # 15
+    #    (260, 261, 266),  # 16
+    #    (348, 349, 262),  # 17
+    #    (233, 234, 235),  # 18
+    #    (237, 238, 239),  # 19
+    #    (339, 340, 232),  # 20
+    #    (327, 328, 329),  # 21
+    #    (305, 91, 90),  # 22
+    #    (296, 61, 60),  # 23
+    # )
+    ledmap_bytes = bytearray(
```

Comment:
Style note: these don't need to be `bytearray`s; they can just be `bytes`, and you don't even need to say `bytes`:
```py
ledmap_bytes = (
        b"\x01\x1F\x00\x1F\x00\x1E"
        b"\x01\x16\x00\x01\x00\x00"
...
)

Reply:
Just take away the `bytearray` word, that's easy.

Reply:
Ah! Thank you. I’ll make that change along with the RGB order stuff then, will commit again later.